components/DatasetDetails/DatasetAboutInfo.vue

```diff
@@ -109,6 +119,9 @@ export default {
      */
     getRecordsUrl: function() {
       return `${process.env.discover_api_host}/search/records?datasetId=${this.$route.params.datasetId}`
+    },
+    s: function(){
+      return (this.externalPublications.length > 0) ? 's' : ''
```

Comment:
I would recommend something a little more specific to the use case.
```suggestion
    associatedPublicationHeading: function(){
      return (this.externalPublications.length > 0) ? 'Associated Publications' : 'Associated Publication'
```

Reply:
Nice idea @cameronbaney. Will commit this suggestion